go/ocr2/decryptionplugin/queue.go

```diff
@@ -0,0 +1,14 @@
+package decryptionplugin
+
+type CiphertextId = []byte
+
+type DecryptionRequest struct {
+	ciphertextId CiphertextId
+	ciphertext   []byte
+}
+
+type DecryptionQueuingService interface {
+	GetRequests(requestCountLimit int, totalBytesLimit int) []DecryptionRequest
+	GetCiphertext(ciphertextId CiphertextId) ([]byte, error)
+	ReturnResult(ciphertextId CiphertextId, plaintext []byte)
```

Comment:
This was changed a couple times already but maybe "SetResult()" is a better name to follow the set/get pattern?

Reply:
SGTM, @KuphJr ?

Reply:
Sure, I can make this change.